app/Http/Controllers/SongController.php

```diff
@@ -100,20 +95,22 @@ public function destroy(Song $song)
 
         return redirect()->route('songs.index');
     }
-    private function validateNewData()
+    private function validateSongData()
     {
         return request()->validate([
             'title' => 'required',
             'artist' => 'required',
-            'entry' => 'min:3',
-            'emoji' => '',
-            'image' => 'image|mimes:jpeg,png,jpg,gif|max:2048'
+            'track_id' => 'nullable',
+            'artist_id' => 'nullable'
+
         ]);
     }
     private function validateData()
```

Comment:
Maybe choose a more specific name like you did with the validateSongData-Method?

Reply:
I also felt that the method needs a better name, but I couldn't think of one. Maybe I can call it validateSongDetails because it 'validates' optional user input